[es] Translate of README.md to spanish.

Jonakss opened the pull request:
Also added a reference to anothers languages. The links reference to README.<lg>.md in the root of the repository.

Jonakss committed:
Update README.md

Added links to other languages in the main README.md / Se añaden links a los demás lenguajes en el archivo principal README.md

Jonakss committed:
Create README.es.md

Created the translate of REDME.md in spanish. / Creado la traducción al español de README.md.

Jonakss committed:
Fixed typos

Jonakss committed:
Fixed redaction, md issues and typos

Jonakss committed:
Fixed text of languages links

Jonakss committed:
Link in actual language

Jonakss committed:
Use md code to separator "|"

darigovresearch commented:
Hi Jonathan, thank you very much for your pull request!

Could you please make the following small adjustments?

1. Move the line explaining the different available languages to be a new paragraph in the Introduction sections of `readme.md` & `readme.es.md`
2. Change the links to readmes that don't exist instead yet to the relevant issue for that language as they don't exist yet and it will encourage people to make a good translation like you have!
3. Change the words "Machine Learning" to "Aprendizaje Automático" on lines 3 & 10 of your `readme.es.md` file to make it consistent with the Spanish flashcards

Will be happy to merge this pull request and close the relavant issue when this has been done

Jonakss committed:
Changes requested to PR

Move the line explaining the different available languages to be a new paragraph in the Introduction sections of readme.md & readme.es.md
Change the links to readmes that don't exist instead yet to the relevant issue for that language as they don't exist yet and it will encourage people to make a good translation like you have!
Change the words "Machine Learning" to "Aprendizaje Automático" on lines 3 & 10 of your readme.es.md file to make it consistent with the Spanish flashcards

Jonakss committed:
Changed languages avaiables formatting

Jonakss committed:
Fixed formatting

Jonakss commented:
I have added you suggestions. Tell me if you need something else.
Cheers.

Jonakss committed:
Changed Machine Learning to Aprendizaje automático.

darigovresearch commented:
Merge closes #3
